### DISCUSSION TITLE
When will the WixBA be officially shipped in wix v4?

### DISCUSSION THREAD

**Discussion by Bo-Liu-BL** [wixtoolset/issues#7527] [Questions] [answered]
Hello experts,

From the following page:
[https://wixtoolset.org/docs/fourthree/faqs/#upgrading-v3-managed-bootstrapper-applications](https://wixtoolset.org/docs/fourthree/faqs/#upgrading-v3-managed-bootstrapper-applications):
Looks the managed WixBA hasn't been GAed:
```
Even though the WixBA is not currently shipping, it is still available at https://github.com/wixtoolset/wix4/tree/develop/src/test/burn/WixToolset.WixBA.
``` 

Is there any explicit/scheduled roadmap or timeline for the managed BA to be GAed?
Thank you very much!

**Comment by barnson** [comment]
WixBA is a sample, so not something is "released."

**Reply by Bo-Liu-BL** [reply]
Thank you very much for the reply, @barnson .
Does it mean the managed C# BA has been shipped in wix v4?

**Reply by BMurri** [reply, marked as answer]
The interfaces and base classes needed to build your own managed BA as well as the glue code to load a managed BA into a bundle have been shipped (similar to v3). What was not shipped was a bundle installing the toolset itself because the deployment mechanism for v4 is nuget packages, rather than the installed toolset/zip file deployment mechanisms of v3.

**Comment by Bo-Liu-BL** [comment]
Thank you very much for the reply, @barnson and @BMurri .
